UnicornEngine/video/include/unicorn/video/CameraFpsController.hpp

```diff
@@ -0,0 +1,41 @@
+/*
+* Copyright (C) 2017 by Godlike
+* This code is licensed under the MIT license (MIT)
+* (http://opensource.org/licenses/MIT)
+*/
+
+#ifndef UNICORN_VIDEO_CAMERA_FPS_CONTROLLER_HPP
+#define UNICORN_VIDEO_CAMERA_FPS_CONTROLLER_HPP
+
+#include <unicorn/utility/SharedMacros.hpp>
+#include <unicorn/video/CameraController.hpp>
+
+namespace unicorn
+{
+namespace video
+{
+/**
+ * @brief Simple template for quick prototyping game - FPS controller.
+ */
+class CameraFpsController : public CameraController
+{
+public:
+    UNICORN_EXPORT CameraFpsController(std::shared_ptr<Camera> camera);
```

Comment:
Does it have to be shared ownership, wouldn't camera exist even without controller?